portfolio/src/main/webapp/script.js

```diff
@@ -1,28 +1,44 @@
-// Copyright 2019 Google LLC
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     https://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
+/*Handle background-change request */
+async function getRandomImageAsync() {
+    let response = await fetch("/background-change");
+    console.log("Got respone");
+    let imageURL = await response.text();
+    console.log(imageURL);
+    document.getElementById("bg-image").style.background = "linear-gradient(rgba(0,0,0,0.5),rgba(0,0,0,0.5)), url(" + imageURL +") no-repeat fixed center";
+};
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+/*Handle onload for comment section in intro.html*/
+async function getExistingComments() {
+    let response = await fetch("/list-comments");
+    console.log("Got respone");
+    let json = await response.json();
+    console.log(json);
+    let container = document.getElementById("comment-container");
+    container.innerHTML = "";
+    json.forEach(comment => {
+        let node = document.createElement("LI"); 
+        
+        let name = document.createElement("span");
+        name.innerText = "Name: " + comment.name;
+        let content = document.createElement("span");
+        content.innerText = "Content: " + comment.content;
+        
+        let deleteElement = document.createElement("button");
```

Comment:
Nit: I notice that this button is right up against the end of the comment, you might want to add some additional spacing here.

Reply:
This is really weird. I don't think I have getRandomGreeting() function in my script.js file any more.